tests/unit/scanners/test.javascript.js

```diff
@@ -118,6 +117,17 @@ describe('JavaScript Scanner', () => {
     expect(linterMessages.length).toEqual(0);
   });
 
+  it('should support es6 modules via html module and ast-check', async () => {
+    const addonLinter = new Linter({
+      _: ['tests/fixtures/webextension_es6_module.zip'],
```

Comment:
We should also test this on an unpacked extension 
(e.g. with the HTML parsing part of this pull request, we would need to verify that the code that compute the module paths works correctly for unpacked extension, because it may be affected by the difference with the platform filesystem, e.g. the different separator used on windows and the "case insensitivity" of the osx filesystem).

Do you mind to include the fixture test extension unpacked in the pull request? 
(which would also make it easier to review the content of the fixture extension, and to change it in future pull requests if we need to cover some additional scenarios around the es6 modules) 
and then pack it as a zip as part of the test setup to verify the behaviors for a packed extension submitted on AMO.

Reply:
Will do!